Code/Editor/CryEdit.h

```diff
@@ -174,7 +180,9 @@ AZ_POP_DISABLE_DLL_EXPORT_BASECLASS_WARNING
     virtual bool InitInstance();
     virtual int ExitInstance(int exitCode = 0);
     virtual bool OnIdle(LONG lCount);
-    virtual CCryEditDoc* OpenDocumentFile(const char* lpszFileName);
+    virtual CCryEditDoc* OpenDocumentFile(const char* lpszFileName,
+        bool addToMostRecentFileList=true,
+        COpenSameLevelOptions openSameLevelOptions = COpenSameLevelOptions::NotReopenIfSame);
```

Comment:
Thanks for renaming `bAddToMRU` to `addToMostRecentFileList`, it's so much understandable this way.

Now that you're changing this code you could also name `lpszFileName` to `filename`.